endpoints.tf

```diff
@@ -54,6 +54,7 @@ data "template_file" "remote_metastores_yaml" {
     metastore_host   = "${lookup(data.external.endpoint_dnsnames.*.result[count.index],"dnsname")}"
     metastore_port   = "${lookup(var.remote_metastores[count.index],"port")}"
     mapped_databases = "${ lookup(var.remote_metastores[count.index],"mapped-databases","") }"
+    whitelist        = "${ lookup(var.remote_metastores[count.index],"whitelist","") }"
```

Comment:
Can we rename to `writable_whitelist` if this is intended to whitelisting database to write?